tests/momento/cache_client/test_sorted_set_simple.py

```diff
@@ -0,0 +1,131 @@
+"""Simple Sorted Set Tests.
+
+The tests here were developed to quickly verify some code fixes in a straightforward manner.
+The rspec type test setup was making it essentially impossible for me to understand how to
+actually test fetching sorted sets by score in a way that actually triggered the code branch
+I was trying to hit.
+
+We can certainly take another stab at figuring out the "right" way to do this in the rspec
+style setup, or we can continue to add simplified supplemental tests like this for cases where
+the test harness is obtuse and burdensome.
```

Comment:
I'm fully on board with this. @malandis I'm interested in your opinion on this, not so much for this PR but for the general idea of treating this as the pattern for tests in this repo as we move forward from here.

Reply:
These are a lot easier to grok and the code is plenty dry.